### PR TITLE
[14.0][MIG] hr_employee_age

### PR DESCRIPTION
**Pull request by komsan-S**
Standard migration

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot : 

image

**Comment by luistorresm** [comment]
Hello @komsan-S 

Could you squash administrative commits?

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

> Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot. IMPORTANT: Don't squash legit translation commits, authored by their translators, with the message "Translated using Weblate (...)".

**Comment by komsan-S** [comment]
@luistorresm Squash finished 
Thank you :)

**Review by luistorresm — APPROVED** [review]
👍

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

File: hr_employee_age/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    age = fields.Integer(string="Age", readonly=True, compute="_compute_age")
```
Comment (astirpe) [inline hr_employee_age/models/hr_employee.py]: `string="Age"` and `readonly=True` are the default for this field:

```suggestion
    age = fields.Integer(compute="_compute_age")
```

File: hr_employee_age/tests/test_hr_employee.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestHrEmployee(common.TransactionCase):
+    def setUp(self):
+        super(TestHrEmployee, self).setUp()
```
Comment (astirpe) [inline hr_employee_age/tests/test_hr_employee.py]:
```suggestion
        super().setUp()
```

**Review by astirpe — APPROVED** [review]
Just minor nitpickings

File: hr_employee_age/views/hr_employee.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="arch" type="xml">
+            <field name="birthday" position="after">
+                <field name="age" />
```
Comment (tobiaszehntner) [inline hr_employee_age/views/hr_employee.xml:9]: In my opinion, `Age 0` should not be displayed if the birthday hasn't been set
```suggestion
                <field name="age" attrs="{'invisible': [('birthday', '=', False)]}" />
```

**Comment by tobiaszehntner** [comment]
@luistorresm can this be merged? (preferably with the proposed changes from the reviewers applied). We need it for a client.

File: hr_employee_age/views/hr_employee.xml (hunk 3 above)
Comment (kittiu) [reply hr_employee_age/views/hr_employee.xml:9]: Only be this one, I don't change. Hope you are ok. Just don't want to change the current behavior compare to previous version :).

**Comment by kittiu** [comment]
Rebased

@Saran440 can you merge this please?

File: hr_employee_age/tests/test_hr_employee.py (hunk 2 above)
Comment (kittiu) [reply hr_employee_age/tests/test_hr_employee.py]: Fix

**Pull request merged**